h/api/storage.py

```diff
@@ -61,12 +74,77 @@ def create_annotation(request, data):
     :returns: the created annotation
     :rtype: dict
     """
-    annotation = elastic.Annotation(data)
-
-    # FIXME: this should happen when indexing, not storing.
-    _prepare(request, annotation)
+    if not request.feature('postgres_write'):
+        return _legacy_create_annotation_in_elasticsearch(request, data)
```

Comment:
This is fixed now, see https://github.com/hypothesis/h/pull/3153#issuecomment-205929233